### PR TITLE
DM-45856: Add new verification pipelines for IsrTaskLSST to _ingredients.

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

File: pipelines/_ingredients/verifyBiasLSST.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      - analyzeBiasCore
+tasks:
+  verifyBiasIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "verifyBiasIsrExp"
+      connections.outputBin1Exposure: "verifyBiasIsrExpBin8"
+      connections.outputBin2Exposure: "verifyBiasIsrExpBin64"
+      doBias: true
```
Comment (czwa) [inline pipelines/_ingredients/verifyBiasLSST.yaml]: Same ordering comment as on `cp_pipe`.  As-applied would be my preference, but alphabetical at least makes it faster to find a particular option.

File: pipelines/_ingredients/verifyDefectsLSST.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "verifyDefectsIsrExp"
+      doDefect: true
+      doBias: true
+      doDark: true
+      doFlat: true
+      doDiffNonLinearCorrection: false
+      doBootstrap: false
+      # TODO: Add deferred charge and brighter-fatter pipelines
```
Comment (czwa) [inline pipelines/_ingredients/verifyDefectsLSST.yaml]: Ticket number?

File: pipelines/_ingredients/verifyDefectsLSST.yaml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      doBrighterFatter: false
+      doDeferredCharge: false
+      doLinearize: true
+      doCorrectGains: false
+      doApplyGains: true
+      doVariance: true
+      doSaturation: false
+      doSuspect: false
+      doCrosstalk: true
```
Comment (czwa) [inline pipelines/_ingredients/verifyDefectsLSST.yaml]: Does this need the quadratic crosstalk terms enabled?  I think those are disabled by default.

**Review by czwa — APPROVED** [review]
Same kind of questions as on the cp_pipe PR.

File: pipelines/_ingredients/verifyDefectsLSST.yaml (hunk 3 above)
Comment (erykoff) [reply pipelines/_ingredients/verifyDefectsLSST.yaml]: Ooh, good point.

**Pull request merged**